src/sets.jl

```diff
@@ -438,6 +438,34 @@ Base.isapprox(a::SOS2{T}, b::SOS2{T}; kwargs...) where T = isapprox(a.weights, b
 
 dimension(s::Union{SOS1, SOS2}) = length(s.weights)
 
+"""
+	ActivationCond
+
+Activation condition for an indicator constraint.
+Used as first type parameter of `IndicatorSet{ActiveCond,S}`.
+"""
+@enum ActivationCond begin
+    ACTIVATE_ON_FALSE
```

Comment:
Use `ZERO` and `ONE` instead of `FALSE` and `TRUE`. We don't overload the meaning of 0/1 variables with bools anywhere else in the MOI.